projects/storefrontapp/src/app/spartacus/features/b2b-digital-payments/b2b-digital-payments.module.ts

```diff
@@ -0,0 +1,8 @@
+import { NgModule } from '@angular/core';
+import { CheckoutB2BModule } from '@spartacus/checkout/b2b';
+import { DigitalPaymentsModule } from '@spartacus/digital-payments';
+
+@NgModule({
+  imports: [DigitalPaymentsModule, CheckoutB2BModule],
+})
+export class B2BDigitalPaymentsModule {}
```

Comment:
Didn't we agree to have a "wrapper" module only for _one_ checkout? Either the base, b2b _or_ repl?

I think it's enough if just keep the `projects/storefrontapp/src/app/spartacus/features/digital-payments-feature.module.ts`